app/portainer/components/stack-duplication-form/stack-duplication-form.js

```diff
@@ -7,6 +7,7 @@ angular.module('portainer.app').component('stackDuplicationForm', {
     onMigrate: '&',
     endpoints: '<',
     groups: '<',
-    currentEndpointId: '<'
+    currentEndpointId: '<',
+    applicationState: '<'
```

Comment:
I do not think there is a need to bind `applicationState` at all here. Please remove any occurence of this binding.